android/src/main/AndroidManifest.xml

```diff
@@ -1,9 +1,12 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-  package="de.julianassmann.flutter_background">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="de.julianassmann.flutter_background">
   <uses-permission android:name="android.permission.WAKE_LOCK" />
   <uses-permission android:name="android.permission.FOREGROUND_SERVICE"/>
   <uses-permission android:name="android.permission.REQUEST_IGNORE_BATTERY_OPTIMIZATIONS" />
+  <uses-permission android:name="android.permission.FOREGROUND_SERVICE_SPECIAL_USE" />
   <application>
-  <service android:name="de.julianassmann.flutter_background.IsolateHolderService" android:exported="true" /> 
+  <service
+      android:name="de.julianassmann.flutter_background.IsolateHolderService"
+      android:foregroundServiceType="specialUse"
```

Comment:
Users might want to customize this.

I'd suggest leave out the `<service>` and ask users to set it themselves.